sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -268,7 +272,15 @@ object ScalaNativePluginInternal {
       discover("clang++", Seq(("3", "8"), ("3", "7")))
     },
     nativeClangOptions := {
-      includes ++ libs ++ maybeInjectShared(nativeSharedLibrary.value)
+      val target = Seq("-target", nativeTarget.value)
+      target ++ includes ++ libs ++ maybeInjectShared(
+        nativeSharedLibrary.value)
+    },
+    nativeTarget := {
+      Process("llc -version").lines_!.collectFirst {
+        case line if line.contains("Default target: ") =>
+          line.replace("Default target: ", "").replace(" ", "")
+      }.get
```

Comment:
This is not working out that well. Apparently there is more than one name for the same target on MacOS depending on whether apple-built or homebrew-built clang is used. We should probably just compile dummy c file with `-S -emit-llvm` and find `target triple` in there. This will also remove dependency on `llc`.